include/networkit/centrality/LocalPartitionCoverage.hpp

```diff
@@ -42,7 +42,7 @@ class LocalPartitionCoverage : public Centrality {
      * The name of this algorithm.
      * @return "Local partition coverage"
      */
-    std::string toString() const override;
+    TLX_DEPRECATED(std::string toString() const override);
```

Comment:
That will not make a difference in terms of functionality, but shouldn't it be like`std::string  TLX_DEPRECATED(toString() const override);`? This of course applies to all occurences.